### ISSUE TITLE
Datagrid Pagination - pagination 'Total Pages' label and count is not provided to screen reader

### ISSUE THREAD

**Issue by sumegha26** [vmware/clarity#4739]
## Describe the bug

- Total Pages label and total pages count is not provided to the screen reader
- When current page input text box is focused, screen reader reads it as `Current page` and number
- Total Pages label is a  `` <span> </span> `` which is not focusable and has aria-label.

Which has aria-label as 'Total Pages', as it is not focusable screen reader does not read that. Needs a way to read Total Pages

## How to reproduce

https://clarity.design/documentation/datagrid/pagination

1.  Go to 'https://clarity.design/documentation/datagrid/pagination'
2.  Navigate pagination using Tab.
3.  Screen reader reads Current Page and given number
4.  Screen reader does not read Total Pages and count

## Expected behavior

 Screen reader should read Total Pages and count

**Comment by sumegha26** [comment]
- Adding unique id on `total pages`  ``<span id="unique"> {{page.last}} </span> ``  and reference this id by adding `aria-describedby` on  `current page` input box should read it.

When input box is focused, screen reader will read `Current Page` label and current page number and by `aria-describedby` on input box  it will read `Total Pages` label and total pages  number.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
